File: app/src/main/java/mariavv/fitnesspal/model/entity/Dish.java
package mariavv.fitnesspal.model.entity;

import java.util.Date;

public class Dish {
    private Date date;
    private int foodId;
    private int mass;

    public Dish(Date date, int foodId, int mass) {
        this.date = date;
        this.foodId = foodId;
        this.mass = mass;
    }

    public Date getDate() {
        return date;
    }

    public int getFoodId() {
        return foodId;
    }

    public int getMass() {
        return mass;
    }
}

File: app/src/main/java/mariavv/fitnesspal/data/db/jounal/Dish.kt
package mariavv.fitnesspal.data.db.jounal

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "dishes")
class Dish(
        @PrimaryKey(autoGenerate = true)
        @ColumnInfo(name = "id")
        var id: Int = 0,
        @ColumnInfo(name = "meal")
        var meal: String,
        @ColumnInfo(name = "date")
        var date: Long,
        @ColumnInfo(name = "food_id")
        var foodId: Int,
        @ColumnInfo(name = "weight")
        var weight: Int
)